src/ASM/ASMbase.h

```diff
@@ -560,6 +560,14 @@ class ASMbase
   virtual bool injectNodeVec(const Vector& nodeVec, Vector& globVec,
 			     unsigned char nndof = 0, int basis = 0) const;
 
+  //! \brief Extracts nodal results for this patch from the global vector.
```

Comment:
Extracts --> Injects og from --> into
globVec er [out] (eller egentlig både in og out) og nodeVec er [in]

Dessuten, list gjerne parametrene i samme rekkefølge som i metode-deklarasjonen. Og madof kunne da være argument nr. 3 slik at det eneste som skiller denne metoden fra den andre med samme navn, er nettopp arg nr. 3.